cmd/main.go

```diff
@@ -57,8 +57,12 @@ func init() {
 		nsxOperatorNamespace = os.Getenv("NSX_OPERATOR_NAMESPACE")
 	}
 
-	if cf.EnableHA == false {
+	if cf.EnableHA == nil || *cf.EnableHA == true {
```

Comment:
Maybe we can warp this  EnableHA initialization into a function and put in config.go, and then we can use UT to test this function.

Reply:
Moreover, in the future, I'm thinking we also need follow this way to initialize the config options in a a central place like in config.go instead of in main